api/cfm-openapi.yaml

```diff
@@ -1630,3 +1658,15 @@ components:
         description:
           type: string
           description: The description of service(s) offered by the URI
+    device:
+      type: object
+      properties:
+        name:
+          type: string
+        address:
+          type: string
+        port:
+          type: integer
+        type:
+          type: string
```

Comment:
on line 1672, **_add_**:
`enum: [blade, cxl-host]`

Note here that I've changed `cma` to `blade`.
I'm not sure of the consequences of this for your code.
Please have a look.
The reason for this change is that, right now, in PoC3, from a CFM user perspective, we are adding blades, **_not_** CMAs.  We need to hide the implementation details (ie: the "cma" search string) from the user.  Right now, a "cma" represents a blade.  In the future, on a centralized BMC system, "cma" will truly represent an entire appliance.  However, we need to present to the CFM user what the code can do today, not what it might do in the future.

Reply:
Hi, Scott, I prefer to use `cma`, since we use `cma` as txt in blade servers(`txt = ["cma=true"]`). And we can just use cma to filter the blade servers out of cxl-host servers, otherwise we need to translate `blade` to `cma` in the code.

Reply:
Yes, I know; and for the reasons I explained in my previous post.
I'm asking how difficult the translation from "blade" to "cma" would be?
I do not think it's a good idea to expose the setting "cma" to the service client when it's not actually a cma, it's a blade.